### ISSUE TITLE
[Feature Request] Carthage Compatibility

### ISSUE THREAD

**Issue by bennokress** [grafele/PullToReach#4]
I would like to include PullToReach in one of my projects which works with Carthage instead of CocoaPods. Sadly right now this is not possible with the following log when running `carthage update --platform ios`:

```
/usr/bin/xcrun xcodebuild -workspace /Users/Benno/Projekte/Privat/Spielebuch/Carthage/Checkouts/PullToReach/Example/PullToReach.xcworkspace -scheme PullToReach -configuration Release -derivedDataPath /Users/Benno/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/PullToReach/0.1.1 -sdk iphoneos ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES archive -archivePath /var/folders/4r/8hw9fr5551n7xmffrbsmmc8m0000gn/T/PullToReach SKIP_INSTALL=YES GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=NO CLANG_ENABLE_CODE_COVERAGE=NO STRIP_INSTALLED_PRODUCT=NO (launched in /Users/Benno/Projekte/Privat/Spielebuch/Carthage/Checkouts/PullToReach)User defaults from command line:
    IDEArchivePathOverride = /var/folders/4r/8hw9fr5551n7xmffrbsmmc8m0000gn/T/PullToReach
    IDEDerivedDataPathOverride = /Users/Benno/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2.1_10E1001/PullToReach/0.1.1

Build settings from command line:
    CARTHAGE = YES
    CLANG_ENABLE_CODE_COVERAGE = NO
    CODE_SIGN_IDENTITY = 
    CODE_SIGNING_REQUIRED = NO
    GCC_INSTRUMENT_PROGRAM_FLOW_ARCS = NO
    ONLY_ACTIVE_ARCH = NO
    SDKROOT = iphoneos12.2
    SKIP_INSTALL = YES
    STRIP_INSTALLED_PRODUCT = NO

2019-05-25 18:59:12.694 xcodebuild[35699:10086722]  DTDeviceKit: deviceType from d592f435c9aa7f98dcc641fd142d947ea2cb182f was NULL
2019-05-25 18:59:12.738 xcodebuild[35699:10086717]  DTDeviceKit: deviceType from d592f435c9aa7f98dcc641fd142d947ea2cb182f was NULL
note: Using new build system
note: Planning build
note: Constructing build description
Build system information
error: An empty identity is not valid when signing a binary for the product type 'Application'. (in target 'PullToReach_Example')

** ARCHIVE FAILED **
```

It seems like the example project is the problem right now, there might be more errors down the line as soon as it gets over this, though.

**Comment by grafele** [comment]
Hi @bennokress . I've updated the project and released version 0.1.2. The carthage problem should be solved now.

**Comment by bennokress** [comment]
that was fast, thanks :)
